hedera-node/src/main/java/com/hedera/services/state/submerkle/CurrencyAdjustments.java

```diff
@@ -45,15 +42,15 @@
 	static final int MAX_NUM_ADJUSTMENTS = 25;
 
 	long[] hbars = NO_ADJUSTMENTS;
-	List<EntityId> accountIds = Collections.emptyList();
+	long[] accountCodes = NO_ADJUSTMENTS;
```

Comment:
Since we are using `long[]`, can name this `accountNums`.

Reply:
Renamed !